projects/ngrx-auto-entity/src/lib/factories/util.ts

```diff
@@ -0,0 +1,32 @@
+import { ActionCreator, Creator } from '@ngrx/store';
+import { EntityAction } from '../..';
+import { TNew } from '../actions/model-constructor';
+
+export interface CorrelatedProps {
+  correlationId?: string;
+}
+
+export interface StandardProps extends CorrelatedProps {
+  criteria?: any;
+}
+
+export const NAE_TYPE_ACTION_CACHE = '__nae_type_action_cache';
```

Comment:
This constant should be exported from the public interface, so that end developers may access the action factory cache dynamically if they need to.